colima/protos/colima.proto

```diff
@@ -85,11 +85,12 @@ message SgxQuote {
 }
 
 message Program {
-  bytes code = 1;
+  string file_name = 1;
+  bytes code = 2;
 }
 
 message Data {
-  uint32 package_id = 1;
+  string file_name = 1;
```

Comment:
Ah! You may have already thought of my previous comment and are moving in that direction.